etc/analyzetests/main.go

```diff
@@ -0,0 +1,154 @@
+// Package main is a go test analyzer that publishes results to a MongoDB database.
```

Comment:
for these tests/coverage commands, I'm assuming no changes were made from the copy in RDK?